test/e2e/predictor/test_triton.py

```diff
@@ -85,6 +85,7 @@ def test_triton_runtime():
                 name="pytorch",
             ),
             runtime="kserve-tritonserver",
+            protocol_version="v2",
```

Comment:
since tritonserver runtime is specified, do you still need to set the `protocol_version`?

Reply:
yes @yuzisun.. even though we provide runtime, we validate model format against runtime's supported model format.. the protocol version should be added in order to satisfy.. At the end of validation, we are adding defaults..